test/jdk/java/awt/Window/MainKeyWindowTest/TestMainKeyWindow.java

```diff
@@ -29,6 +29,7 @@
  * @bug 8194327
  * @summary [macosx] AWT windows have incorrect main/key window behaviors
  * @author Alan Snyder
```

Comment:
I guess we need to remove @author tag

Reply:
I have left it to make credit an external contributor